### ISSUE TITLE
App is crashing with class 10 when using a the 6th perk from top when clicking on the start menu

### ISSUE THREAD

**Issue by ZRenegade** [North101/ghplayertracker#17]
Hi,

So my group and I are using this app for our remote games and we encounter a bug when using class 10 with a certain perk ("mod_class_10_muddle_rolling").

To reproduce :
1.  Add the class 10
2. Check the 6th perk (once or twice did not matter).
3. Save
4. Click on the sword menu (Start)

Result : The app crash.

Could you please give me any pointer as to where to look at to fix this?

Btw, I'm willing to fix others issues too, I'm just new to Gradle (and Git in general too).

**Comment by North101** [comment]
https://github.com/Aarhun/ghplayertracker/releases
Aarhun has continued developing this app. I recommend checking out their version and see if it fixes your issue

**Comment by ZRenegade** [comment]
@North101 Thanks for the info. I'll look into it. Does this mean this project is no longer active nor will release any future version?

**Comment by North101** [comment]
Yep. And if I was going to continue it I'd probably completely rewrite it

**Comment by ZRenegade** [comment]
Well thanks for the app and thanks for your time!

**Comment by North101** [comment]
No prob. Glad you like it and hope the other one fixes your issue
